playgrounds/petclinic-maven-sb-2/src/main/resources/templates/owners/ownersList.html

```
@@ -0,0 +1,61 @@
+      <tr>
+        <th th:text="#{name}" style="width: 150px;">Name</th>
+        <th th:text="#{address}" style="width: 200px;">Address</th>
+        <th th:text="#{city}">City</th>
+        <th th:text="#{telephone}" style="width: 120px">Telephone</th>
+        <th th:text="#{pets}">Pets</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr th:each="owner : ${listOwners}">
+        <td>
+          <a th:href="@{/owners/__${owner.id}__}" th:text="${owner.firstName + ' ' + owner.lastName}" /></a>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </a> ]

(tag-pair)